### ISSUE TITLE
Dynamic sizing

### ISSUE THREAD

**Issue by eddow** [kveeiv/extjs-boxselect#20]
Sizing the box after rendering leads to chaotic size of the input-text : the size of the input is set in 'beginLayoutFixed' where the sizes we have are the size _before_ the re-sizing.

**Comment by eddow** [comment]
Dudes, I'm really sorry : trying to debug the application from your website to spot the differences, I saw there was a new version and it seems the bug has been solved from 2.0.1 to 2.0.3 .

My bad.

**Comment by eddow** [comment]
So, everything is better but I still found some troubles in the vertical sizing this time. I c/p here the change of beginLayoutFixed I did for it to work better :

```
beginLayoutFixed: function(ownerContext, width, suffix) {
    var me = this,
        owner = ownerContext.target;

    owner.triggerEl.setStyle('height', '24px');

    me.callParent(arguments);

    /*No inputElCt calculations here!*/
},

publishInnerHeight:function(ownerContext, height) {
    var owner = ownerContext.target;

    if (ownerContext.heightModel.fixed /*&& ownerContext.lastBox*/) {
        owner.listWrapper.setStyle('height', height+'px');
        owner.itemList.setStyle('height', '100%');
    }
},
```
